package.json

```diff
@@ -16,7 +16,7 @@
     "test:android": "cd native/androidTest && ./gradlew connectedAndroidTest",
     "test:ios": "scripts/test-ios",
     "test:native": "concurrently -n android,ios 'npm run test:android' 'npm run test:ios' --kill-others-on-fail",
-    "test:typescript": "cd examples/typescript; yarn; npm run test",
+    "test:typescript": "yarn build; cd examples/typescript; yarn; npm run test",
```

Comment:
hmm, building is necessary?

Reply:
Because I setup the package as a link (https://github.com/Nozbe/WatermelonDB/pull/1477/files#diff-52a1428e31d738b6bcc8a9c9c840fa66484d5bf4e84921e0628eeb7596148371R9), I chose to use the production output as it best represents the final folder stucture.

I could map it to source though, but eslint will complains (although typescript _should_ work)

Reply:
I had tried it here: https://github.com/Nozbe/WatermelonDB/pull/1477/commits/b04a356af5907b97cb502ecd2ebc925e99df5637

Reply:
If possible, I'd rather not run `yarn build` on `yarn ci:check`. Having `dist/` causes issues with other tools :/

Reply:
@radex I disabled eslint for the ts example app and made it work using typescript paths in [597e3fe](https://github.com/Nozbe/WatermelonDB/pull/1477/commits/597e3fe2b0c0e408e7a9fe969331d98b3af6ebde)